game/src/Levels/Level1.tscn

```diff
@@ -102,8 +102,8 @@ gravity_point = true
 position = Vector2( 308.292, 985.772 )
 
 [node name="Exit" parent="." instance=ExtResource( 4 )]
-position = Vector2( 576.472, 793.699 )
+position = Vector2( 368, 1081.39 )
```

Comment:
This exit portal is not configured correctly (no next level) and happens to be exactly where the player spawns, causing a crash when LevelLoader.trigger is called with a Null value.

Reply:
Hmm I think this is what was causing #176 then